SeQuant/core/parse/deparse.cpp

```diff
@@ -190,6 +190,7 @@ std::wstring deparse_expr(Sum const& sum, bool annot_sym) {
 }  // namespace details
 
 std::wstring deparse_expr(ExprPtr expr, bool annot_sym) {
+  if (!expr) return {};
```

Comment:
```suggestion
  if (!expr) return {};
  
```
(for easier grasping different parts of the code)